e2e/testdata/fn-eval/error-in-pipe/.expected/config.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+exitCode: 1
+stdErr: |2
+  [FAIL] "gcr.io/kpt-fn/set-namespace:v0.1"
+    Results:
+      [ERROR] failed to configure function: input namespace cannot be empty
+    Stderr:
+      "failed to configure function: input namespace cannot be empty"
+    Exit code: 1
+
+   
+  [FAIL] "gcr.io/kpt-fn/dne"
+  error: Function image "gcr.io/kpt-fn/dne" doesn't exist.
```

Comment:
This seems inconsistent compared to above. I understand it's the orchestrator throwing the error, but we should think of a way to present this better.

Reply:
I think we should address all these issue in #1915

Reply:
There is another issue https://github.com/GoogleContainerTools/kpt/issues/2084 which is relative to this error message.